doc/source/user_guide/indexing.rst

```diff
@@ -933,6 +933,24 @@ and :ref:`Advanced Indexing <advanced>` you may select along more than one axis
 
    df2.loc[criterion & (df2['b'] == 'x'), 'b':'c']
 
+.. warning::
+
+   ``iloc`` supports two kinds of boolean indexing. If the indexer is a boolean ``Series``,
+   an error will be raised. For instance, in the following example, ``df.iloc[sr.values, 1]`` is ok.
+   This type of boolean indexing is array. But ``df.iloc[sr, 1]`` would raise ``ValueError``.
```

Comment:
"is array" -> "is using an array". Alternatively, "The boolean indexer is an array."
sr -> s

Reply:
Done.